core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -221,11 +223,11 @@ class Invoker(
         implicit transid: TransactionId): Unit = {
         error(this, errorMsg)
         val msg = tran.msg
-        val name = EntityName(actionDocInfo.id().split(EntityPath.PATHSEP)(1))
+        val name = EntityPath(actionDocInfo.id.id)
```

Comment:
Yes, we agreed, that we leave the name in the WhiskActivation as EntityName.
But I changed the type, that is passed to `makeWhiskActivation()` from EntityName to EntityPath, to pass only one variable around and not the name and the fqdn.
At the place, where the name is written to the activation, this name is still EntityName.
I only add the fqdn to the annotations.